### ISSUE TITLE
#345 - GetEMoney() invoked with 1 parameter, 3 required.

### ISSUE THREAD

**Issue by williamdes** [Mangopay/mangopay2-php-sdk#355]
Introduced by: #345

https://github.com/Mangopay/mangopay2-php-sdk/blob/c86023dd160a76999ef1e598d428d126fd31f9c0/MangoPay/ApiUsers.php#L345

Could you please fix this because you released a version https://github.com/Mangopay/mangopay2-php-sdk/releases/tag/2.12.2 breaking semver since the parameters are not optionally defined

This change also breaks phpstan

```
  Line   app/Services/Mangopay/xxx.php                            
 ------ ------------------------------------------------------------------- 
  109    Method MangoPay\ApiUsers::GetEMoney() invoked with 1 parameter, 3  
         required.
``` 

Thanks :)

cc @mickaelpois

**Comment by williamdes** [comment]
@mickaelpois did you see this issue ?

**Comment by mickaelpois** [comment]
Fixed on 2.13.0

**Comment by williamdes** [comment]
Thank you @mickaelpois 
- The CHANGELOG entries are missing

I will wait for the tag to be pushed ;) 


Ref: #363
